Validation/RecoTrack/src/MTVHistoProducerAlgoForTracker.cc

```diff
@@ -1107,7 +1109,25 @@ void MTVHistoProducerAlgoForTracker::bookRecoHistos(DQMStore::IBooker& ibook,
   histograms.assoc_chi2_vs_pt.push_back(makeProfileIfLogX(
       ibook, useLogPt, "assoc_chi2mean_vs_pt", "mean #chi^{2} vs p_{T}", nintPt, minPt, maxPt, 0., 20.));
   histograms.assoc_chi2prob_vs_pt.push_back(makeProfileIfLogX(
-      ibook, useLogPt, "assoc_chi2prob_vs_pt", "mean #chi^{2} probability vs p_{T}", nintPt, minPt, maxPt, 0., 20.));
```

Comment:
here the only _change_ in the plots I've done (max Y limit from 20 to 1, because it's a probability. I thought it was a typo from another PR copying from the line above). If I am wrong I change it back immediately.

Reply:
I agree with this update, thanks